fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,567 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "CCT Customer Service Requests",
+    "description": "This section provides an overview of all aspects of the application programmers’ interface, including registration to use the interface, the various resources that constitute the interface and how these should be orchestrated into an application that will deliver consistent and reliable results.",
+    "contact": {
+      "email": "cct@cct.co.za"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "2.1"
+  },
+  "servers": [
+    {
+      "url": "https://qaeservices1.capetown.gov.za/coct/api/zcur-guest/login",
+      "description": "FCSA Service Request API"
+    }
+  ],
+  "security": [
+    {},
+    {
+      "cookieAuth": []
+    }
+  ],
+  "tags": [
+    {
+      "name": "service_request_group",
+      "description": "This process shows the flow of actions in which the application takes for the creation of a service request. \n"
+    }
+  ],
+  "paths": {
+    "/coct/api/zcur-guest/login": {
+      "get": {
+        "tags": [
+          "service_request_group"
```

Comment:
I suggest adding this to an `auth` tag

Reply:
Auth Tag has been added

Reply:
Ah, I meant that this particular path should be tagged `auth` rather than `service_request_group`